services/search/pkg/config/config.go

```diff
@@ -18,7 +18,7 @@ type Config struct {
 
 	GRPC GRPC `yaml:"grpc"`
 
-	Datapath string `yaml:"data_path" env:"SEARCH_DATA_PATH" desc:"Path for the search persistence directory."`
+	Datapath string `yaml:"data_path" env:"SEARCH_DATA_PATH" desc:"The directory where the filesystem storage will store search data. If not definied, the root directory defaults to $OCIS_BASE_DATA_PATH:/search."`
```

Comment:
the documentation automatically populates the default column: https://owncloud.dev/services/search/configuration/
```suggestion
	Datapath string `yaml:"data_path" env:"SEARCH_DATA_PATH" desc:"The directory where the filesystem storage will store search data."`
```

Same for the other changes. We shouldn't repeat the default here manually, when we can automate that, right?

Reply:
Disagree, see alternative suggestion and comment from @dragonchaser